lib/process-services-cloud/src/lib/group/components/group-cloud.component.ts

```diff
@@ -81,26 +80,26 @@ export class GroupCloudComponent implements OnInit, OnChanges {
 
     /** Emitted when a group is selected. */
     @Output()
-    selectGroup: EventEmitter<GroupModel> = new EventEmitter<GroupModel>();
+    selectGroup: EventEmitter<IdentityGroupModel> = new EventEmitter<IdentityGroupModel>();
 
     /** Emitted when a group is removed. */
     @Output()
-    removeGroup: EventEmitter<GroupModel> = new EventEmitter<GroupModel>();
+    removeGroup: EventEmitter<IdentityGroupModel> = new EventEmitter<IdentityGroupModel>();
 
     @ViewChild('groupInput')
     private groupInput: ElementRef<HTMLInputElement>;
 
-    private selectedGroups: GroupModel[] = [];
+    private selectedGroups: IdentityGroupModel[] = [];
 
-    private searchGroups: GroupModel[] = [];
+    private searchGroups: IdentityGroupModel[] = [];
 
-    private searchGroupsSubject: BehaviorSubject<GroupModel[]>;
+    private searchGroupsSubject: BehaviorSubject<IdentityGroupModel[]>;
 
-    private selectedGroupsSubject: BehaviorSubject<GroupModel[]>;
+    private selectedGroupsSubject: BehaviorSubject<IdentityGroupModel[]>;
 
-    searchGroups$: Observable<GroupModel[]>;
+    searchGroups$: Observable<IdentityGroupModel[]>;
```

Comment:
Just make this a `BehaviorSubject<IdentityGroupModel[]>` and remove the private fields. BehaviorSubject is already an Observable.

Reply:
fixed